codespell_lib/data/dictionary.txt

```diff
@@ -23295,6 +23314,7 @@ relevence->relevance
 relevent->relevant
 relfected->reflected
 reliablity->reliability
+relie->rely, relies, relieve, really,
```

Comment:
Relieve seems unlikely to me, or have you seen it in the real world?

Reply:
ATM I do not recall and am not sure where I saw it. Nevertheless, I've swapped `relieve` with `relief`